src/shed/__init__.py

```diff
@@ -75,6 +75,8 @@
     # https://github.com/astral-sh/ruff/issues/10245
     # ruff replaces `sorted(reversed(iterable))` with `sorted(iterable)`
     # "C414", # unnecessary-double-cast # codemod: `replace_unnecessary_nested_calls`
+
+
     # ** These are new fixes that Zac had enabled in his branch
```

Comment:
I initially only enabled checks that had a direct analog in autoflake/pyupgrade/isort/codemods, but that left a bunch that you had enabled in your branch. Presumably you added them because you thought they were useful to have enabled by default?

Reply:
@Zac-HD ... soooo should I do a separate PR where I do enable them?

Reply:
Open an issue and I'll get to in eventually?

The more important one to me personally is that our just-released shed no longer detects hypothesis' own test suite as first-party, and so all the imports get moved.

None of this is work-related though!